docs/admins.rst

```diff
@@ -573,6 +576,11 @@ registry_post_replace
   replacing namespace/repo, all ``old`` registries in image pullspecs will be
   replaced by their ``new`` replacements.
 
+skip_all_allow_list
+  List of koji packages (with comment about The Vault Exception ID) which are
```

Comment:
What is `Vault Exception ID` from upstream POW?

Reply:
@lcarva do you know?

Reply:
How about we omit `(with comment about The Vault Exception ID)` from the upstream documentation completely? This is an internal business requirement.

Reply:
removed